File: selectPhoto/src/main/java/com/jy/selectphoto/bean/TopTitleBean.java
package com.jy.selectphoto.bean;

import java.io.Serializable;

/**
 */

public class TopTitleBean implements Serializable {
    public String firstPath;
    public String name;
    public int num;

    public TopTitleBean(int num, String firstPath, String name) {
        this.num = num;
        this.firstPath = firstPath;
        this.name = name;
    }
}

File: library/src/main/java/com/ximalife/library/http/model/TopTitleBean.kt
package com.ximalife.library.http.model

class TopTitleBean {
    var firstPath: String? = null
    var name: String? = null
    var num = 0

    constructor(firstPath: String?, name: String?, num: Int) {
        this.firstPath = firstPath
        this.name = name
        this.num = num
    }
}